packages/fast-layouts-react/src/column/column.tsx

```diff
@@ -170,7 +171,9 @@ export class Column extends Foundation<ColumnHandledProps, ColumnUnhandledProps,
                 : this.props.order;
         }
 
-        const gridStyles: React.CSSProperties = canUseCssGrid()
+        const canUseCssGridStyle: boolean =
+            this.props.cssGridPropertyName === "grid" ? true : canUseCssGrid();
```

Comment:
Do you think it's unintuitive that this still has the potential to generate CSS grid properties even if I'm explicitly setting the `cssGridPropertyName` to `"-ms-grid"`. Probably not a common scenario, but I think this should mimic the handled/unhandled pattern of form controls where, if set, `cssGridPropertyName` would always be respected. Thoughts?

Reply:
Yeah that's a fair point, I'll update to

    const canUseCssGridStyle: boolean =
          this.props.cssGridPropertyName === "grid" ? true :
          this.props.cssGridPropertyName === "-ms-grid" ? false : 
          canUseCssGrid();